scripts/gulpfiles/build_tasks.js

```diff
@@ -430,15 +447,15 @@ return ${exportsExpression};
  * @return {{chunk: !Array<string>, js: !Array<string>}} The chunking
  *     information, in the same form as emitted by
  *     closure-calculate-chunks.
- *
- * TODO(cpcallen): maybeAddClosureLibrary?  Or maybe remove base.js?
  */
 function getChunkOptions() {
   if (argv.compileTs) {
     chunks[0].entry = path.join(TSC_OUTPUT_DIR, chunks[0].entry);
   }
```

Comment:
I should have deleted this in this PR.  Fortunately noticed by an external contributor and corrected in #6431.